FuGetGallery.csproj

```diff
@@ -23,7 +20,7 @@
     <PackageReference Include="ListDiff" Version="1.0.7" />
   </ItemGroup>
   <ItemGroup>
-    <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="2.0.2" />
+    <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="2.0.4" />
```

Comment:
Tool references are deprecated in favor of the newer .NET Core 3.x architecture for dotnet tools. If you want to use this tool for development, you can install it now using `dotnet tool install --global dotnet-aspnet-codegenerator`.

But this reference in the project file can be removed.

Reply:
Thanks; didn't know this. Looks like it came from [the initial commit](https://github.com/praeclarum/FuGetGallery/commit/0ad719883d2093986b912e5b487edb38f2cf3961#diff-91315fb1368747ad8910a3717a2007f951b6393ac0884f27578399329f7d7f4f) but can be cleaned up now.